src/User32/User32.cs

```diff
@@ -3647,5 +3569,27 @@ public static partial class User32
            IntPtr hMenu,
            IntPtr hInstance,
            void* lpParam);
+
+        /// <summary>
+        /// Changes an attribute of the specified window. The function also sets a value at the specified
+        /// offset in the extra window memory.
+        /// </summary>
+        /// <param name="hWnd">A handle to the window and, indirectly, the class to which the window belongs.</param>
+        /// <param name="nIndex">The zero-based offset to the value to be set.</param>
+        /// <param name="dwNewLong">The replacement value.</param>
+        /// <returns>
+        /// <para>If the function succeeds, the return value is the previous value of the specified offset.</para>
+        /// <para>If the function fails, the return value is zero. To get extended error information, call GetLastError.</para>
+        /// <para>If the previous value is zero and the function succeeds, the return value is zero, but the function does
+        /// not clear the last error information. To determine success or failure, clear the last error information by
+        /// calling SetLastError with 0, then call SetWindowLongPtr. Function failure will be indicated by a return value of
+        /// zero and a GetLastError result that is nonzero.</para>
+        /// </returns>
+        /// <remarks>
+        /// When compiling for 32-bit Windows, SetWindowLongPtr is defined as a call to the SetWindowLong function. This
+        /// function is exposed using a helper that conditionally calls SetWindowLong in 32-bit processes.
+        /// </remarks>
+        [DllImport(nameof(User32), SetLastError = true, EntryPoint = nameof(SetWindowLongPtr))]
```

Comment:
I don't think we should use `nameof(SetWindowLongPtr)` for the `EntryPoint` here. That's the name of the helper method, and if we ever renamed that method, it would not be correct to change the string here.

Reply:
Done. Also, I added another definition as part of the latest commit (AdjustWindowRectEx). Just filling in the gaps as I find 'em.